src/edit_virtual_machine.zig

```diff
@@ -881,3 +888,93 @@ fn set_buffer(buffer: []u8, value: []const u8) void {
         index += 1;
     }
 }
+
+fn delete_confirmation_modal() !void {
+    // Render a modal when deleting a VM
+    var confirmation_window = try gui.floatingWindow(@src(), .{
+        .modal = true,
+        .open_flag = &deleting_vm,
+    }, .{
+        .color_style = .window,
+        .min_size_content = .{ .w = 300, .h = 100 },
+    });
+    defer confirmation_window.deinit();
+
+    var confirmation_vbox = try gui.box(@src(), .vertical, .{
+        .expand = .both,
+        .padding = .{ .x = 20, .y = 20, .w = 20, .h = 20 },
+    });
+    defer confirmation_vbox.deinit();
+
+    // Render some text to confirm deletion
+    try gui.label(@src(), "Are you sure you want to delete this virtual machine?", .{}, .{
+        .expand = .horizontal,
+    });
+
+    // Confirmation text input
+    {
+        var confiration_input_vbox = try gui.box(@src(), .vertical, .{
+            .expand = .both,
+            .padding = .{ .x = 0, .y = 5, .w = 0, .h = 5 },
+        });
+        defer confiration_input_vbox.deinit();
+
+        try gui.label(@src(), "Enter the full name of the VM to confirm deletion:", .{}, .{
+            .expand = .horizontal,
+            .font = .{
+                .size = 8,
+                .name = "VeraIt",
+                .ttf_bytes = fonts.VeraIt,
+            },
```

Comment:
Don't specify the font manually. Note that you'll be able to get rid of this fonts const "alias" as well.

Reply:
How would I specify an italic font? It's not in the DVUI theme at all.

Reply:
Ah, well in that case you could open an issue on DVUI's GitHub repository ot contribute that functionality yourself. I'll resolve for now.